torchchat/export.py

```diff
@@ -39,6 +39,7 @@ def export_for_server(
     output_path: str = "model.pt2",
     dynamic_shapes: bool = False,
     package: bool = True,
+    metadata: Optional[Dict[str, str]] = None,
```

Comment:
```suggestion
    metadata: Dict[str, str] = {},
```

We can delete `metadata = metadata or {} `

Reply:
general advice is to not have mutable structures as default args because they survive invocations =>
https://docs.python-guide.org/writing/gotchas/